packages/sync-actions/src/product-actions.js

```diff
@@ -434,10 +451,26 @@ function _buildSetAttributeAction (
   return action
 }
 
-function _buildVariantImagesAction (diffedImages, oldVariant, newVariant) {
+function _buildVariantImagesAction (
+  diffedImages,
+  oldVariant = {},
+  newVariant = {},
+) {
   const actions = []
-
+  // generate a hashMap to be able to reference the right image from both ends
+  const imagesHashMap = findMatchingPairs(
```

Comment:
Can we just call this `matchingImagePairs` to be descriptive around what's inside not what it is?